jj/mock/_mocked.py

```diff
@@ -40,6 +42,11 @@ async def fetch_history(self) -> List[HistoryItem]:
         self._history = await self._handler.fetch_history()
         return self._history
 
+    @property
+    def parsed_history(self) -> Union[List[HistoryItem], List[str], None]:
```

Comment:
For methods that call other functions instead of just returning variables, it's better to design them as regular methods rather than using getter-style naming

```python
def get_formatted_history(self):
    ...
```